wiki/People/Beatmap_Nominators/en.md

```diff
@@ -45,6 +45,17 @@ Modders that aim to apply to the Beatmap Nominators must fulfil the following cr
 - The gamemode (osu!, osu!mania, osu!catch, osu!taiko) that you modded beatmaps in does not matter as your activity in the modding community as a whole is being measured and not the activity in a certain game mode.
 - The point in time at which the Kudosu has been awarded to your mod post **does** matter, as the calculator embed in the website will fetch the data from your own Modding History events.
 
+### Modding expectations
+
+Submitted mods should show that a user is capable of analyzing and improving maps to a standard fit for the Beatmap Nominators.
+
+The following are modding traits expected of Beatmap Nominators. Failure to display any trait may result in a rejected application.
+- Coverage of all difficulty levels
+- Identification of unrankable issues, including ones tools can't detect like timing/metadata
+- Comparisons of parts of a map to support issues/suggestions
+- A wide variety of map elements, such as rhythm, spacing, movement, intensity, contrast, and consistency.
```

Comment:
```suggestion
- Commentary about a wide variety of map elements, such as rhythm, spacing, movement, intensity, contrast, and consistency
```
not sure if commentary is the best word but i couldnt think of anything better immediately